Legacy table for anatomical MRI data (#58)
bids-standard/bep001: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

| T1 Rho map         | T1rho            | Quantitative T1rho brain imaging<br>[http://www.ncbi.nlm.nih.gov/pubmed/24474423](http://www.ncbi.nlm.nih.gov/pubmed/24474423) <br> [http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4346383/](http://www.ncbi.nlm.nih.gov/pubmed/24474423) |

> ##### LEGACY suffixes (to be deprecated)

Reviewer: ```suggestion
##### Legacy suffixes (to be deprecated)
```

Some suffixes were defined in the original BIDS 1.0.0 - 1.2.0 specification, but it is

> recommended not to use them any more, because they are inconsistent and/or ambiguous.

Reviewer: ```suggestion
RECOMMENDED not to use them any more, because they are inconsistent and/or ambiguous.
```

They will most likely be deprecated in later main versions of BIDS.

However, they are still part of the BIDS 1.x specification for legacy reasons.

> Thus, the following suffixes are VALID, but NOT RECOMENDED:

Reviewer: ```suggestion
Thus, the following suffixes are valid, but SHOULD NOT be used for future BIDS datasets:
```
